src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -445,7 +445,7 @@ private function sendTokenEmail(UserInterface $user, string $from, string $to, s
             ->setSubject($this->getSubject())
             ->setFrom($from)
             ->setTo($to)
-            ->setBody($this->getMessage($user, $token));
+            ->setBody($this->getMessage($user, $token), 'text/html');
```

Comment:
If we change this to `text/html` the email template should be updated I think to use `<br/>` instead of a normal typehint.